tests/test_file_utils.py

```diff
@@ -88,28 +87,22 @@ def gen_random_output():
         self.assertGreater(np.abs(out1 - out3).sum(), 0)
 
 
-def test_zstd_extractor(zstd_path, tmp_path, text_file):
-    input_path = zstd_path
-    assert ZstdExtractor.is_extractable(input_path)
-    output_path = str(tmp_path / "extracted.txt")
-    ZstdExtractor.extract(input_path, output_path)
-    with open(output_path) as f:
-        extracted_file_content = f.read()
-    with open(text_file) as f:
-        expected_file_content = f.read()
-    assert extracted_file_content == expected_file_content
-
-
-@pytest.mark.parametrize("compression_format", ["xz", "zstd"])
-def test_cached_path_extract(compression_format, xz_file, zstd_path, tmp_path, text_file):
-    path = {"xz": xz_file, "zstd": zstd_path}
-    input_path = path[compression_format]
+@pytest.mark.parametrize(
+    "compression_format, expected_text_path_name", [("gzip", "text_path"), ("xz", "text_file"), ("zstd", "text_file")]
+)
+def test_cached_path_extract(
+    compression_format, expected_text_path_name, text_gz_path, xz_file, zstd_path, tmp_path, text_file, text_path
```

Comment:
You can also add a gz_file fixture if you want. This way you don't have to deal with different text path names

Reply:
Done